test/parallel/test-npm-version.js

```diff
@@ -14,5 +14,5 @@ const npmPathPackageJson = path.resolve(
 );
 
 const pkg = require(npmPathPackageJson);
-assert(pkg.version.match(/^\d+\.\d+\.\d+$/),
+assert(pkg.version.match(/^\d+\.\d+\.\d+-rc\.\d+$/),
```

Comment:
```suggestion
assert(pkg.version.match(/^\d+\.\d+\.\d+(-rc\.\d+)?$/),
```

?

Reply:
I don't think we want to do this. In general we don't want to be landing rc's on master or accidentally putting them out in releases... if I recall that was the reason for this test in the first place 😂. As we are going to cut 7.0.0 before Node.js cuts v15.0.0 I was working under the assumption that this test would get updated again when we do the update.